DayrSuryan/DSEthiop13.xml

```diff
@@ -48,14 +52,17 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                   <msItem xml:id="ms_i1">
                         <locus from="1r" to="32r"/>
                      <title type="incomplete" ref="LIT1979Mashaf"/>
-                        <textLang mainLang="gez"/>
-                        <note> Chants for individual
-                      feasts are separated from each other by line of red and black dots
-                      (black starting from f. 23r).</note>
-                     <notatedMusic>
+                        <textLang mainLang="gez" otherLangs="am"/>
+                    <notatedMusic>
                         <desc>The manuscript contains extensive musical notation,
-                          which has not been transcribed here. </desc>
+                          which has not been transcribed here.</desc>
                      </notatedMusic>
+                     <note>The manuscript seems to be an unfinished compilation of excerpts of the work, beginning regularly with
+                        <date ref="ethiocal:Maskaram1">1 Maskaram</date> commemorating <persName ref="PRS5741Johnthe"/>, but continuing
+                        immediately with the commemoration on <date ref="ethiocal:Hedar7">7 Ḫǝdār</date> of Qʷǝsqʷām.
```

Comment:
I would leave out indication to zemmare (I know it from a different context, it will not be clear); for the rest is ok. I  think the text was copied with some omissions. It would be good to quote at least the first few words of the portions for the last two feasts, at least , in a "Note" or somehow ( i see the portion in EMIP01019_MihurGedam_044 frame 9 left), and also to say that the dates - - which were supposed to be rubricated - were inserted after fol. 23. Therefore, we cannot say the exact dates (It appears that some versions of Ziq have non-linear organisation of the feasts.) I would reduce the time range to 1900 - 1950. Small, nasty manuscript!

Reply:
( i see the portion in EMIP01019_MihurGedam_044 frame 9 left) - sorry, disregard this.

Reply:
By the last two feasts, do you mean Giyorgis and Qwesqwam on 21v? The end of the manuscript seems differently organised than the beginning

Reply:
Qwesqwam and Giyorgis are included completely in the explicit